Formula/packmol.rb

```diff
@@ -24,9 +24,12 @@ class Packmol < Formula
   end
 
   def install
+    # Avoid passing -march=native to gfortran
+    inreplace "Makefile", "-march=native", ENV["HOMEBREW_OPTFLAGS"] if build.bottle?
```

Comment:
Trying out `HOMEBREW_OPTFLAGS`. This does use `ENV["HOMEBREW_OPTFLAGS"] == nil` on Apple Silicon which gets `inreplace`'d as empty string, e.g.
```
 ------------------------------------------------------
 Compiling packmol with /opt/homebrew/opt/gcc/bin/gfortran
 Flags: -O3 --fast-math  -funroll-loops
 ------------------------------------------------------
```

Don't know what arch optimization is used here.